tls/s2n_tls13_handshake.c

```diff
@@ -105,25 +105,28 @@ int s2n_tls13_compute_pq_hybrid_shared_secret(struct s2n_connection *conn, struc
     notnull_check(client_ecc_params);
 
     DEFER_CLEANUP(struct s2n_blob ecdhe_shared_secret = { 0 }, s2n_blob_zeroize_free);
+    struct s2n_blob *pq_shared_secret;
```

Comment:
nit: `struct s2n_blob *pq_shared_secret = { 0 }; `

Reply:
this is a pointer, `= NULL` is probably better ;)